triage/views.py

```diff
@@ -190,6 +211,14 @@ def get_context_data(self, **kwargs):
         context['article_group_read_progress'] = (
             self.article_read_manager.get_view_progress_for_groups()
         )
+        if self.triage_answers['sector'] is not None:
```

Comment:
if `sector` is somehow an empty string then the code below will not be hit. would it be valid to instead check if `sector` is falsey with `if self.triage_answers['sector']:` ?

Reply:
no because then you get a key error. i could check for both